### PR TITLE
Rename package-test container to singlehost-test container

### PR DESCRIPTION
**Pull request by johnbieren**
I believe it is misleading for both the integration and package level tests to all come from a container named "package-test" so this PR is simply renaming it for clarity's sake.

Signed-off-by: Johnny Bieren <jbieren@redhat.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -49,7 +49,7 @@ properties(
                                 string(defaultValue: 'stable', description: 'Tag for rpmbuild image', name: 'RPMBUILD_TAG'),
                                 string(defaultValue: 'stable', description: 'Tag for rsync image', name: 'RSYNC_TAG'),
                                 string(defaultValue: 'stable', description: 'Tag for ostree-compose image', name: 'OSTREE_COMPOSE_TAG'),
-                                string(defaultValue: 'stable', description: 'Tag for package test image', name: 'PACKAGE_TEST_TAG'),
+                                string(defaultValue: 'stable', description: 'Tag for package test image', name: 'SINGLEHOST_TEST_TAG'),
```
Comment (scoheb) [inline Jenkinsfile]: nit: change to single host test?

**Comment by johnbieren** [comment]
[test]

**Comment by johnbieren** [comment]
[test]

**Comment by johnbieren** [comment]
[merge]

**Pull request merged**